packages/web/src/components/organisms/ToggleTheme.tsx

```
@@ -0,0 +1,34 @@
+
+/** The themes list. */
+const themes = [VALUE_LIGHT, VALUE_DARK] as const;
+
+/**
+ * The toggle to change the theme.
+ * @returns The component.
+ */
+export const ToggleTheme: Component = () => {
+  const isDarkMode = createDarkMode({ dark: VALUE_DARK, light: VALUE_LIGHT });
+  let ref!: HTMLInputElement;
+  onMount(() => (ref.checked = isDarkMode()));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)